core/domain/prod_validation_jobs_one_off.py

```diff
@@ -26,23 +26,48 @@
 import python_utils
 
 (
-    activity_models, audit_models,
-    classifier_models, collection_models,
-    config_models, email_models, exp_models,
-    feedback_models, improvements_models, job_models,
-    opportunity_models, question_models,
-    recommendations_models, skill_models, stats_models,
-    story_models, suggestion_models, topic_models,
-    user_models,) = (
-        models.Registry.import_models([
-            models.NAMES.activity, models.NAMES.audit,
-            models.NAMES.classifier, models.NAMES.collection,
-            models.NAMES.config, models.NAMES.email, models.NAMES.exploration,
-            models.NAMES.feedback, models.NAMES.improvements, models.NAMES.job,
-            models.NAMES.opportunity, models.NAMES.question,
-            models.NAMES.recommendations, models.NAMES.skill,
-            models.NAMES.statistics, models.NAMES.story,
-            models.NAMES.suggestion, models.NAMES.topic, models.NAMES.user]))
+    activity_models,
```

Comment:
FYI I'm not sure that this is that much more readable, tbh. Maybe you can do it like we do in Angular -- just make sure that each line contains the same deps, so that line are comparable with each other? It's easier to compare 5 lines than 16 lines.

Ditto elsewhere.

Reply:
I'm fine with both I guess. I think the benefit of this is that you when you need to add/remove something you just remove/add two lines. When you have it on fewer lines and you need to remove/add stuff you will need to move the other stuff too.

Reply:
True, but I think reading the list is going to be an order of magnitude more common than modifying it.

Reply:
Hmm, true, but I think that this pattern of having `some_models` is so common in the codebase that I assume that `some_models` will be a reference to `some_models` module that I don't even have to check these imports, thus the actual reading time of these is reduce (at least for me) by skipping it.

Reply:
Anyway, done, as I do not have a strong opinion about this.